packages/wallets/src/features/cashier/modules/WithdrawalVerification/WithdrawalVerificationRequest/WithdrawalVerificationRequest.tsx

```diff
@@ -9,24 +11,26 @@ type TProps = {
 const WithdrawalVerificationRequest: React.FC<TProps> = ({ sendEmail }) => {
     return (
         <div className='wallets-withdrawal-verification-request'>
-            <div className='wallets-withdrawal-verification-request-icon'>
-                <EmailVerification />
-            </div>
-            <div className='wallets-withdrawal-verification-request__messages'>
-                <p className='wallets-withdrawal-verification-request__title'>
-                    Please help us verify your withdrawal request.
-                </p>
-                <p className='wallets-withdrawal-verification-request__description'>
-                    <span>
+            <WalletsActionScreen
+                actionText='Send email'
+                // TODO: Add line breaks
+                description={
+                    <>
                         Click the button below and we’ll send you an email with a link. Click that link to verify your
                         withdrawal request.
-                    </span>
-                    <span>This is to protect your account from unauthorised withdrawals.</span>
-                </p>
-            </div>
-            <button className='wallets-withdrawal-verification-request__send-button' onClick={sendEmail}>
-                Send email
-            </button>
+                        <br />
```

Comment:
Can't we do this using css instead of <br/>?

Reply:
According to the figma reference the description is one whole text section with no specified gap in between and remains constant for desktop/responsive, which is why it was done using `br`

<img width="623" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/1ce89bd6-27bd-45fd-a52a-f4759c925968">